src/screens/BeerOfTheDay/BeerOfTheDay.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { PureComponent } from 'react';
+import { fetchRandomBeer } from '../../services/beers';
+import Heading from '../../shared/components/Heading/Heading';
+import RandomBeer from '../../shared/components/RandomBeer/RandomBeer';
+import Button from '../../shared/components/Button/Button';
+import './styles.scss';
+
+class BeerOfTheDayComponent extends PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      beer: null,
+    };
+  }  
+    getRandomBeer = async () => {
+      const randomBeer = await fetchRandomBeer();
+      this.setState({
+        beer: randomBeer,
+      })
+    }
+  
+
+  render() {
+    const { beer } = this.state;
+    return (
+      <div className="beer-day-page">
+        <Heading 
+          type="h3"
```

Comment:
You're exporting heading types from the component so use them everywhere.

Reply:
Done

Reply:
Done!